plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -954,7 +954,7 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 				    self->dynamic_ts.bonding);
 			return FALSE;
 		}
-		if (self->static_ts.tool_string_version < TOOL_STRING_VERSION_BONDING_QC)
```

Comment:
That condition is never met as `self->static_ts.tool_string_version` is at least `0x30` (aka `'0'`) and `TOOL_STRING_VERSION_BONDING_QC` is `2`.

Reply:
Yes, you are right.

Reply:
cool, thanks.